client/incus_instances.go

```diff
@@ -2616,6 +2620,10 @@ func (r *ProtocolIncus) ConsoleInstanceDynamic(instanceName string, console api.
 		return nil, nil, fmt.Errorf("The server is missing the required \"console_vga_type\" API extension")
 	}
 
+	if console.Force && !r.HasExtension("console_force") {
+		return nil, nil, fmt.Errorf("The server is missing the required \"console_force\" API extension")
```

Comment:
I would write this with back ticks to avoid the need for escaping:
```suggestion
		return nil, nil, fmt.Errorf(`The server is missing the required "console_force" API extension`)
```